gulp/build.js

```diff
@@ -14,8 +14,6 @@ gulp.task('build', function(cb) {
 gulp.task('build-dev', function(cb) {
   runSequence(
     'dist',
-    'compile-all',
-    'dist',
```

Comment:
It makes build-dev much faster. If we want to compile, then we could do
compile-all.

Bowen

On Thursday, April 14, 2016, Jiaming Dong <notifications@github.com> wrote:

> In gulp/build.js
> <https://github.com/ViDA-NYU/genotet/pull/163#discussion_r59819281>:
>
> > @@ -14,8 +14,6 @@ gulp.task('build', function(cb) {
> >  gulp.task('build-dev', function(cb) {
> >    runSequence(
> >      'dist',
> > -    'compile-all',
> > -    'dist',
>
> Is it right to delete these?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/genotet/pull/163/files/62e6bd683662111f2d2051e8d94f7dcd6609b2db#r59819281>
>